channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -261,7 +264,7 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                 let diagLog = _self.diagLog();
                 _evtNamespace = mergeEvtNamespace(createUniqueNamespace("Sender"), core.evtNamespace && core.evtNamespace());
                 _offlineListener = createOfflineListener(_evtNamespace);
-
+                _core = core;
```

Comment:
nit: We actually don't need to "save" this here as `_self.core` (part of the base class) gets initialized with the core instance, this in combination with another suggestion below would remove the need for this.